src/crimson/os/seastore/seastore.h

```diff
@@ -200,12 +200,46 @@ class SeaStore final : public FuturizedStore {
   }
 
 
+  template <typename Ret, typename F>
+  auto repeat_with_onode(
```

Comment:
Hmm, not sure if there is any trick to combine `repeat_with_onode(...) const` with `repeat_with_onode(...)`.